.gitlab-ci.yml

```diff
@@ -25,8 +25,9 @@
   artifacts:
     reports:
       dotenv: deploy.env
+  variables:
+    NODE_ENV: production
```

Comment:
`gitlab-ci` doesn't merge variables in different templates. 

```
deploy:sls:branch:
  <<: *tmpl_branch_deploy_rules
  <<: *tmpl_deploy
```

The result of this template usage is 
```
deploy:sls:branch:
  variables:
    SLS_STAGE: $CI_COMMIT_REF_SLUG
  variables:
    NODE_ENV: production
```

To use variables from multiple templates https://docs.gitlab.com/ee/ci/yaml/#yaml-anchors-for-variables needs to be used.